pkg/server/grpc/tracee.go

```diff
@@ -19,6 +19,574 @@ import (
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
+// Event translation maps - internal to external Event ID
+var EventTranslationMap = [10200]pb.EventId{
```

Comment:
Shouldn't the size of this array be 10,000? Any event id higher than 10000 should be considered non built-in event

Reply:
You are right, think the size of this array should be 10,000, but today when we defined the following line, the value of `events.Stat` will depend on each arch we are compiling it.
```
events.Stat: pb.EventId_stat,
```
if amd64, `events.Stat` will be 4
if arm64, `events.Stat` will be 10,001 (because is [unsupported](https://github.com/aquasecurity/tracee/blob/baaf585ae06e9d19028739fc90f00e9f7859501e/pkg/events/core_arm64.go#L354-L357))

So I think the unsupported range would be below 10,000, e.g: 9,000. In this way will be perfect possible to define array size of 10,000.